app/templates/Gruntfile.js

```diff
@@ -313,53 +336,68 @@ module.exports = function (grunt) {
           src: [
             '*.{ico,txt}',
             'images/{,*/}*.{webp,gif}',
-            'styles/fonts/{,*/}*.*',<% if (sassBootstrap) { %>
-            'bower_components/bootstrap-sass-official/assets/fonts/bootstrap/*.*'<% } %>
+            'styles/fonts/{,*/}*.*',
+<% if (sassBootstrap) { -%>
+            'bower_components/bootstrap-sass-official/assets/fonts/bootstrap/*.*'
+<% } -%>
           ]
         }, {
           src: 'node_modules/apache-server-configs/dist/.htaccess',
           dest: '<%%= yeoman.dist %>/.htaccess'
         }]
       }
-    },<% if (includeRequireJS) { %>
+    },
+<% if (includeRequireJS) { -%>
     bower: {
       all: {
         rjsConfig: '<%%= yeoman.app %>/scripts/main.js'
       }
-    },<% } %><% if (templateFramework === 'mustache') { %>
+    },
+<% } -%>
+<% if (templateFramework === 'mustache') { -%>
     mustache: {
       files: {
         src: '<%%= yeoman.app %>/scripts/templates/',
         dest: '.tmp/scripts/templates.js',
-        options: {<% if (includeRequireJS) { %>
+        options: {
+<% if (includeRequireJS) { -%>
           prefix: 'define(function() { this.JST = ',
-          postfix: '; return this.JST;});'<% } else { %>
+          postfix: '; return this.JST;});'
+<% } else { -%>
           prefix: 'this.JST = ',
-          postfix: ';'<% } %>
+          postfix: ';'
+<% } -%>
         }
       }
-    }<% } else if (templateFramework === 'handlebars') { %>
+    }
+<% } else if (templateFramework === 'handlebars') { -%>
     handlebars: {
       compile: {
         options: {
-          namespace: 'JST'<% if (includeRequireJS) { %>,
-          amd: true<% } %>
+          namespace: 'JST',
```

Comment:
In this case, it can be avoided by moving `amd` above `namespace`, right? Besides from that I don't see big of a problem with such a *same line* tag once upon a time.